### PR TITLE
refactor: consolidate .gitignore files

### LINKED ISSUE #67: Consolidate .gitignore files on Github
Move all rules into root directory .gitignore file.

### PR DESCRIPTION
**Pull request by humayunalamHO**
Resolves #67

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,23 @@
 *.iml
 *.ipr
 /.idea/
+
+# Other
+monitoring-as-code/monitoring-config/output
+
```
Comment (michaelpearsonHO) [inline .gitignore:16]: Remove blank line

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,23 @@
 *.iml
 *.ipr
 /.idea/
+
+# Other
```
Comment (michaelpearsonHO) [inline .gitignore:14]: Can we say this is the # monitoring-as-code artefacts

**Review by michaelpearsonHO — CHANGES_REQUESTED** [review]
Added comments

**Review by michaelpearsonHO — APPROVED** [review]
Approved.

**Review by mahrufiqbalHO — APPROVED** [review]

**Pull request merged**